src/responder/common/negcache.c

```diff
@@ -920,7 +909,7 @@ errno_t sss_ncache_prepopulate(struct sss_nc_ctx *ncache,
         }
     }
 
-    filter_set = false;
+    /* Populate domain-specific negative cache group entries */
     for (dom = domain_list; dom; dom = get_next_domain(dom, 0)) {
```

Comment:
I don't think we should do that there.

The reason for that is that, as far as I understand, filter_users and filter_groups can also be set per-domain in sssd.conf. Considering that I'm not mistaken and that we do not expose this options to subdomains**, I don't think we weed to descend here.

**: here is @mzidek-rh's answer of my question about this on #sssd:
```
10:51 <fidencio> mzidek: are filter_users and filter_groups exposed to subdomains? or those can be only set for domains themselves?
10:51 <fidencio> mzidek: maybe a different way to ask would be ... can I have "filter_users = foo" under a subdomain config section?
10:56 <mzidek> fidencio: no, filter_users and groups is not available in the trusted domain section
10:57 <fidencio> mzidek: okay, thanks!
```

Let me know if you think it should descend to subdomains as well.